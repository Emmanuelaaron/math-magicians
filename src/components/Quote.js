import React from 'react';

const Quote = () => (
  <>
    <section className="home1">
      <div className="content1">
        <p>
          Mathematics is not about numbers, equations,
          computations or algorithms. It is about understanding - William Paul Thurston
        </p>
      </div>
    </section>
  </>
);

export default Quote;
