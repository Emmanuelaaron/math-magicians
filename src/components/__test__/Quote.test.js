import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom/extend-expect';
import Quote from '../Quote';

describe('Quote', () => {
  it('should render the quote component', () => {
    const quote = renderer.create(<Quote />);

    expect(quote).toMatchSnapshot()
  });
});