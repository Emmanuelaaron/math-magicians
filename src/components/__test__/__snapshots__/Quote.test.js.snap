// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should render the quote component 1`] = `
<section
  className="home1"
>
  <div
    className="content1"
  >
    <p>
      Mathematics is not about numbers, equations, computations or algorithms. It is about understanding - William Paul Thurston
    </p>
  </div>
</section>
`;
